test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -12860,6 +12860,24 @@ def add_config(self, conf={}, hup=True):
 
         return self.apply_config(conf, hup)
 
+    def unset_mom_config(self, name, hup=True):
```

Comment:
@hirenvadalia If you check the design discussion about this change on forum you will find that there is already a ticket [PP-611](https://pbspro.atlassian.net/browse/PP-611) that is related to moving all configurations to qmgr. Please add more to that ticket if you feel like.